lib/order.c: Make the dependency loop messages into debug msgs

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -urN rpm-5.4.0.orig/lib/order.c rpm-5.4.0/lib/order.c
--- rpm-5.4.0.orig/lib/order.c	2010-12-13 17:36:39.000000000 -0600
+++ rpm-5.4.0/lib/order.c	2011-08-10 11:11:43.194686332 -0500
@@ -2170,7 +2170,7 @@
 		const char * dp;
 		rpmlogLvl msglvl = (anaconda || (rpmtsDFlags(ts) & RPMDEPS_FLAG_DEPLOOPS))
 			? RPMLOG_WARNING : RPMLOG_ERR;
-#if defined(RPM_VENDOR_MANDRIVA) /* loop-detection-optional-loglevel */
+#if defined(RPM_VENDOR_MANDRIVA) || defined(RPM_VENDOR_POKY) /* loop-detection-optional-loglevel */
 		// Report loops as debug-level message by default (7 = RPMLOG_DEBUG), overridable
 		msglvl = rpmExpandNumeric("%{?_loop_detection_loglevel}%{?!_loop_detection_loglevel:7}");
 #endif
